### PR TITLE
MOC-153: swap imgs for svgs & fix round buttons

### PR DESCRIPTION
**Pull request by fitzk**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new smaller button variant with enhanced keyboard interaction support.
	- Added a collection of SVG icon components for improved visual consistency across the application.
	- Implemented new logo components for better branding representation.

- **Improvements**
	- Replaced static images with icon components to enhance performance and maintainability.
	- Streamlined the Header and Toast components for improved UI design and readability.
	- Enhanced the RecordButton component for better state handling and user feedback.

- **Bug Fixes**
	- Adjusted class names and properties in various components for consistency and clarity.

- **Styling**
	- Refined the structure and styling of multiple components to enhance user experience and accessibility.

Screenshots show a subset of the changes:

Screenshot 2024-08-05 at 9 38 45 AM
Screenshot 2024-08-05 at 9 38 23 AM

### CODE REVIEW

File: apps/mocksi-lite/common/Logos.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+				fill="#009875"
+			/>
+		</svg>
```
[the remark this answers is not in the document]
Comment (fitzk) [reply apps/mocksi-lite/common/Logos.tsx:13]: I agree code Rabbit, in due time

File: apps/mocksi-lite/common/Icons.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
+export function CloseIcon() {
+	return (
```
Comment (fitzk) [inline apps/mocksi-lite/common/Icons.tsx:2]: Usually we would pass in the colors as props or at least inherit "currentColor" but the icons all look correct right now so in the name of stability let's leave things hardcoded for now

File: apps/mocksi-lite/common/Icons.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+				y="4"
+			/>
+		</svg>
+	);
+}
+
+export function PlayIcon() {
+	return (
+		<svg fill="none" height="22" width="22" xmlns="http://www.w3.org/2000/svg">
+			<title>play icon</title>
```
Comment (elg0nz) [inline apps/mocksi-lite/common/Icons.tsx:102]: nice! Thanks for including `<title>` tags

**Review by elg0nz — APPROVED** [review]
Nice!!
The app looks so much sharper now 😺

**Pull request merged**